server/stree/bench_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2024 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stree
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+)
+
+func BenchmarkSubjectTreeMatch(b *testing.B) {
+	st := NewSubjectTree[struct{}]()
+
+	for i := 0; i < b.N; i++ {
+		// Do 100 objects for 2nd token.
+		subj := fmt.Sprintf("subj.%d.%d", rand.Intn(100)+1, i)
+		st.Insert([]byte(subj), struct{}{})
+	}
+
+	b.Logf("Running with %d subjects", b.N*b.N)
```

Comment:
Minor & purely cosmetic but `b.N*b.N` is no longer correct here now that the second loop was taken out.

Reply:
Actually though I removed that whole file, since I did the other one. Will remove.